src/pages/signup.jsx

```diff
@@ -107,51 +107,47 @@ export default function Signup({ dogs }) {
             placeholder="Name"
             {...register("name")}
             type="text"
-            className= {errors.name ? "py-1 pr-1 pl-3 mb-1 rounded w-full border border-solid border-red-500 border-2 font-maven-pro": 
-            "py-2 pr-1 pl-3 mb-4 rounded w-full border border-solid border-primary-gray border-1 font-maven-pro"}
+            className={`textbox-base text-input ${errors.name ? "textbox-error mb-1" : "textbox-border mb-4"} w-full`}
           ></input>
-          {errors.name && <div className="font-maven-pro text-black font-medium flex items-center"><ExclamationCircleIcon className="h-8 w-8 pr-2"/>{errors.name.message}</div>}
+          {errors.name && <div className="font-maven-pro text-primary-text font-normal text-lg flex items-center mb-1"><ExclamationCircleIcon className="h-8 w-8 pr-2"/>{errors.name.message}</div>}
           
           <input
             id="email"
             placeholder="Email Address"
             {...register("email")}
             type="text"
-            className={errors.email ? "py-1 pr-1 pl-3 mb-1 mt-1 rounded w-full border border-solid border-red-500 border-2 font-maven-pro": 
-            "py-2 pr-1 pl-3 mb-4 rounded w-full border border-solid border-primary-gray border-1 font-maven-pro"}
+            className={`textbox-base text-input ${errors.email ? "textbox-error mb-1" : "textbox-border mb-4"} w-full`}
```

Comment:
I think `font-maven-pro` got left out here, so the font for inputs on the signup page is different than on the login page